dev-lua/sol2/sol2-3.2.2.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-{1..4} luajit )
+inherit cmake lua
+
+DESCRIPTION="Header-only C++ <-> Lua API wrapper"
+HOMEPAGE="https://github.com/ThePhD/sol2"
+SRC_URI="https://github.com/ThePhD/sol2/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+REQUIRED_USE="${LUA_REQUIRED_USE}"
+RESTRICT="!test? ( test )"
+
+RDEPEND="${LUA_DEPS}"
+DEPEND="
+	test? (
+		dev-cpp/catch:1
+		${LUA_DEPS}
+	)
+"
+BDEPEND="
+	test? (
+		virtual/pkgconfig
+	)
+"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-luajit-pkgconf.patch
+	"${FILESDIR}"/${P}-catch-depend.patch
```

Comment:
Was just about to give this a properly look+test, but this patch is missing (I'll have another look after it's there, assuming it's needed).

Reply:
Oops. Added.